package.json

```diff
@@ -48,20 +48,24 @@
     "electron-log": "4.3.2",
     "electron-squirrel-startup": "1.0.0",
     "electron-store": "5.1.1",
+    "emoji-mart": "^3.0.1",
```

Comment:
Could we remove this from emoji selector?
![image](https://user-images.githubusercontent.com/1525725/119606089-e7c85080-bdbf-11eb-969c-7e35ae92ab20.png)

Reply:
Emoji selector doesn't disappear when I click something else. Same for audio selector

Reply:
Also, audio selector doesn't work for me - when I press the "mic" icon, the recording doesn't start.